pydantic_ai_slim/pydantic_ai/profiles/__init__.py

```diff
@@ -35,6 +35,11 @@ class ModelProfile:
     json_schema_transformer: type[JsonSchemaTransformer] | None = None
     """The transformer to use to make JSON schemas for tools and structured output compatible with the model."""
 
+    thinking_start_tag: str = '<think>'
+    """The tag used to indicate thinking parts in the model's output. Defaults to '<think>'."""
+    thinking_end_tag: str = '</think>'
```

Comment:
Can we make this a single field `thinking_tags: tuple[str, str] = ('<think>', '</think>')`, as they'll always be specified together? That we can also pass this value to the methods that use it instead of the entire model profile!